services/wallet/datastore.go

```diff
@@ -71,7 +71,7 @@ func init() {
 // Datastore holds the interface for the wallet datastore
 type Datastore interface {
 	datastore.Datastore
-	LinkWallet(ctx context.Context, ID string, providerID string, providerLinkingID uuid.UUID, anonymousAddress *uuid.UUID, depositProvider, country string) error
+	LinkWallet(ctx context.Context, ID string, providerID string, providerLinkingID uuid.UUID, depositProvider string) error
```

Comment:
Nit: While here, we could tidy it up further:

```suggestion
	LinkWallet(ctx context.Context, id, providerID string, providerLinkingID uuid.UUID, depositProvider string) error
```

Reply:
added to the wallet refactor ticket